plat/kvm/x86/lcpu_start.S

```diff
@@ -38,7 +38,6 @@
 #include <uk/asm.h>
 #include <uk/plat/common/lcpu.h>
 #include <uk/reloc.h>
-#include <x86/cpu_defs.h>
```

Comment:
This one is okay as above it already includes `#include <uk/plat/common/lcpu.h>`, which implicitly includes `uk/arch/lcpu.h`.